test/tests/syntax.json

```diff
@@ -490,6 +490,31 @@
       "src": "{0E-1}",
       "exp": "0E-1"
     },
+    {
+      "description": "- as unquoted-literal",
+      "src": "{-}",
+      "exp": "-"
+    },
+    {
+      "description": ". as unquoted-literal",
+      "src": "{·}",
+      "exp": "·"
+    },
+    {
+      "description": "emoji as unquoted-literal",
+      "src": "{🥔}",
+      "exp": "🥔"
+    },
+    {
+      "description": "multi-glyph emoji as unquoted-literal",
```

Comment:
Multi-glyph is wrong, as it's a single glyph. What you mean is multi _code point_

Reply:
```suggestion
      "description": "multi-code-point emoji as unquoted-literal",
```